app/src/main/AndroidManifest.xml

```diff
@@ -1,336 +1,302 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="com.kickstarter"
-  xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools">
-
-  <!-- PERMISSIONS -->
-
-  <uses-permission android:name="android.permission.INTERNET" />
-  <!--
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Was this file just formatted?

Reply:
- added the deepling intent data on DiscoveryActviity tags
- and <activity ...></activity> has been changed to <activity .... /> for those activities without intent data